setup.cfg

```diff
@@ -1,6 +1,10 @@
 [bdist_wheel]
 universal=1
 
+[codespell]
+ignore-words-list = cant,dne,statics
+skip = cms/forms/widgets.py,*.js,*.po
```

Comment:
`cms/forms/widgets.py,` can be removed after #7300 lands.

Reply:
@cclauss #7300 is merged now. Please go ahead and make this change.